src/pages/Inventory/AddItem.js

```diff
@@ -1,7 +1,119 @@
-import React from "react";
+import React, { Fragment, useState } from "react";
+import withWrapper from "common/withWrapper";
+import Navbar from "common/Navbar";
+import { useHistory } from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+import { useNotification } from "notification";
 
 const AddItem = () => {
-    return <div>I am add itme page</div>;
+    const [data, setData] = useState({
+        name: "",
+        price: "",
+        description: "",
+        date: new Date(),
+        id: Math.floor(Math.random() * 101 + 1),
+    });
+    const history = useHistory();
+    const { triggerNotification } = useNotification();
+
+    const AddItem = (e) => {
+        e.preventDefault();
+        // Adding to local storage
+        if (localStorage.getItem("inventory_data") == null) {
+            localStorage.setItem("inventory_data", "[]");
+        }
+        const inventoryData = JSON.parse(
+            localStorage.getItem("inventory_data")
+        );
+        inventoryData.push(data);
+        localStorage.setItem("inventory_data", JSON.stringify(inventoryData));
+        triggerNotification("Item added successfully", { type: "success" });
+        history.push("/inventory");
+    };
+    return (
+        <Fragment>
+            <Navbar opened="inventory" />
+            <div className="page-content p-5 bg-primary">
+                <div className="page-heading-wrapper mb-5 p-5">
+                    <span className="title">
+                        {" "}
+                        <FormattedMessage id="title.items"></FormattedMessage>{" "}
+                    </span>
+                </div>
+                <div className="card px-5 mx-5" style={{ maxWidth: "400px" }}>
+                    <div className="py-5">
+                        <form onSubmit={(e) => AddItem(e)}>
+                            <div className="form-group mx-5 my-3">
+                                <label className="mb-3">
+                                    {" "}
+                                    <FormattedMessage id="customer.name"></FormattedMessage>
+                                </label>
+                                <input
+                                    type="text"
+                                    name="name"
+                                    required
+                                    value={data.name}
+                                    onChange={(e) =>
```

Comment:
its better to create custom hooks to handle the form
like

`const { handleChange, formData } = useForm();`
something like this